client/src/shared/weNeedYou/index.jsx

```diff
@@ -93,7 +93,7 @@ const WeNeedYou = () => {
 
 	return (
 		<WeNeedYouSection>
```

Comment:
the entire section can hold the onClick event (also please remove it from the button so we don't accidentally mess this up if we ever want to change this).
also please add `cursor: pointer` css rule so the whole thing behaves as if it was a link/button.